### PR TITLE
PocketPY

### PR DESCRIPTION
**Pull request by IridescentRose**
Installs [PocketPy](https://github.com/blueloveTH/pocketpy) for PSP. This is useful for integrating Python scripting into the PSP with C++.

### CODE REVIEW

File: pocketpy/PSPBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    make --quiet
+}
+
+package() {
+    cd "$pkgname/build"
+
+    mkdir -m 755 -p "$pkgdir/psp/lib" "$pkgdir/psp/include" "$pkgdir/psp/include/pocketpy"
+    install -m 644 libpocketpy.a "$pkgdir/psp/lib"
+    install -m 644 ../include/pocketpy.h "$pkgdir/psp/include"
+    install -m 644 ../include/pocketpy/*.h "$pkgdir/psp/include/pocketpy"
```
Comment (sharkwouter) [inline pocketpy/PSPBUILD:26]: Can you also install the license into the package?

**Comment by sharkwouter** [comment]
This is cool! I just found the license was missing, otherwise the package looks good.

File: pocketpy/PSPBUILD (hunk 1 above)
Comment (IridescentRose) [reply pocketpy/PSPBUILD:26]: Yeah -- I'll get that by the end of the day.

**Comment by IridescentRose** [comment]
Updated -- *should* be good?

**Comment by sharkwouter** [comment]
Thanks!

**Pull request merged**